peter.html

```diff
@@ -1 +1,3 @@
 Hi! My name is peter and this is the demo!
+This is a second line of code
+Third!
```

Comment:
The syntax of your codes is outdated....please go to Mozilla ressources for up to date react codes.